mri/operators/fourier/non_cartesian.py

```diff
@@ -494,18 +498,25 @@ def __init__(self, kspace_loc, shape, implementation='cpu', n_coils=1):
     def _op(self, data):
         fft_along_z_axis = np.fft.fftshift(np.fft.fft(
             np.fft.ifftshift(data, axes=2),
-            n=self.acq_num_slices,
```

Comment:
why was this removed?

Reply:
@zaccharieramzi this is removed to have a full FFT, they will use this to later choose what undersampled data they want.